src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -251,6 +252,7 @@ class DistributionConfiguration extends Component {
                                 fieldErrors={this.props.fieldErrors}
                                 self={this}
                                 stateName="channelConfig"
+                                provider={selectedProvider}
```

Comment:
why are we passing provider here? Is this left over from some previous old setup?